src/django_linear_migrations/apps.py

```diff
@@ -106,6 +107,17 @@ def names(self) -> set[str]:
             if not is_pkg and name[0] not in "_~"
         }
 
+    @cached_property
+    def plan(self) -> list[tuple[str, str]]:
+        loader = MigrationLoader(connections[DEFAULT_DB_ALIAS])
+        nodes = [key for key in loader.graph.leaf_nodes() if key[0] in self.app_label]
+        plan = []
+        for node in nodes:
+            for migration in loader.graph.forwards_plan(node):
+                if migration not in plan:
+                    plan.append(migration)
+        return plan
```

Comment:
its just a copy of https://github.com/adamchainz/django-linear-migrations/pull/208 (with passing `at_end=True`)

Reply:
I'd feel more comfortable calling the method in Django, even if it is private.

Reply:
ok, i will use typing.cast to avoid Any